packages/components/src/index.native.js

```diff
@@ -35,6 +35,7 @@ export {
 
 export { default as __experimentalStyleProvider } from './style-provider';
 export { default as BaseControl } from './base-control';
+// @todo review this RN export
 export { hasSplitBorders as __experimentalHasSplitBorders } from './border-box-control/utils';
```

Comment:
I don't know much about our RN setup. If we go with the approach in this PR, should we also remove this and copy it over closer to where the native is? I'm not sure if we should also remove it from RN at this point. I did search throughout GB, and it also looks like it's not being used by RN consumers in the codebase, but I'm not sure if it could be used by other third-party packages.

Reply:
@dcalhoun @WordPress/native-mobile can you please confirm that it's safe to remove this export? Thank you! 🙏

Reply:
👋🏻 Thanks for the ping. 

The mobile unit tests are passing on the CI server, so that is a good sign. 😄 I did not encountering any issues after checking out the proposed changes and "smoke testing" the editor on an iPhone simulator. 

From quickly reviewing the related code, my perception is that `__experimentalHasSplitBorders` is not utilized by the native mobile editor. So, removing this export should likely be safe.